delivery.yaml

```diff
@@ -38,6 +40,10 @@ pipeline:
         if [ "$CUR_PART" != "$VERSION_PART" ]; then NEW_PATCH=0; fi
         RELEASE_VERSION=${VERSION_PART}.${NEW_PATCH}
         export VERSION="${RELEASE_VERSION}"
+
+        echo "Creating git tag: ${RELEASE_VERSION}"
+        git gh-tag "${RELEASE_VERSION}"
```

Comment:
This will create always a tag even if linters fail.

Reply:
This might be ok, at least it would not block subsequent pipelines like it does now but just "waste" a version number

Reply:
I can move it after `make deps cicheck staticcheck gosec` but before docker builds

Reply:
Yes please

Reply:
Done